core/src/main/resources/default-main.conf

```diff
@@ -13,8 +14,13 @@
 # remote.nossl.hosts = *.redhat.com, 10.0.*
 ######################################################################
 
-# remote.nossl.hosts = "10.*,172.*,localhost"
+# Regexes of allowed NON-SSL remote hosts separated by ","
+remote.nossl.hosts = localhost,.apache.org
 
+# Internal Store Validation enabled
+_internal.store.validation.enabled = false
```

Comment:
We should leave this entirely out of the config files, and only use it in AbstractIndyFunctionTest and whatever functional tests we use to specifically test this new feature.

Reply:
Is this means that we are gonna assign "true" to  "_internal.store.validation.enabled" variable each time when we run functional tests?
Also what about running indy instance - is "_internal.store.validation.enabled" variable will be initialized to true in every bean that is gonna use validation for remote stores?
 - something like in @Posconstruct methods or any  overridden init methods?